stable/pipelines/templates/buildplane-config-k8s.yaml

```diff
@@ -0,0 +1,19 @@
+{{- if .Values.buildPlane.dynamic.provider.k8s.enabled }}
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: {{ include "pipelines.fullname" . }}-dynamic-buildplane-config-k8s
+  labels:
+    {{- include "pipelines.labels" . | nindent 4 }}
+data:
+  provider: k8s
+  accountId: {{ default "userId" .Values.buildPlane.dynamic.customer.accountId | quote }}
```

Comment:
I think this should be in the values.yaml

Reply:
no, it does not